### ISSUE TITLE
Create string table for squad names

### ISSUE THREAD

**Issue by L-McKendrick** [Soliders-in-Arms-Arma-3-Group/SIA-Mission-Framework.VR#9]
Reduce amount of redundancy and make it easier to update group names by create a string table for reference https://community.bistudio.com/wiki/Stringtable.xml

**Comment by SiegeToaster** [comment]
I do not think that a string table is necessary for this.  Its main purpose is implementing multiple languages, so if anything, a file should be made to define group names.  However, I do not see what is stopping mission makers from just updating the group names in the eden editor.
